consultation_analyser/consultations/models.py

```diff
@@ -184,6 +184,7 @@ class Answer(UUIDPrimaryKeyModel, TimeStampedModel):
     respondent = models.ForeignKey(Respondent, on_delete=models.CASCADE)
     text = models.TextField()
     chosen_options = models.JSONField(default=list)
+    is_theme_mapping_audited = models.BooleanField(default=False)
```

Comment:
I think we also want to track which user did the auditing - did we decide in the tech meeting last week that we would just use the history (from django-simple-history) to track this in this model?

Reply:
Have done this in a separate PR: https://github.com/i-dot-ai/consult/pull/424.